lib/partition_setup.pm

```diff
@@ -235,6 +295,12 @@ sub addpart {
     send_key(is_storage_ng() ? $cmd{next} : $cmd{finish});
 }
 
+=head2 addvg
+
+ addvg(name => $name [, add_all_pvs => $add_all_pvs]);
+Adds a LVM volume group
```

Comment:
It starts with vowel sound _/ɛl/_

```suggestion
Adds an LVM volume group
```

Reply:
I learned only in front of a real vowel and Hs that are not being spoken?

Reply:
https://dictionary.cambridge.org/grammar/british-grammar/a-an-and-the